tsuru/client/event_test.go

```diff
@@ -554,7 +557,13 @@ func (s *S) TestEventCancel(c *check.C) {
 	trans := &cmdtest.ConditionalTransport{
 		Transport: cmdtest.Transport{Message: runningEvt, Status: http.StatusOK},
 		CondFunc: func(req *http.Request) bool {
-			c.Assert(req.FormValue("reason"), check.Equals, "my reason")
+			var eventCancel tsuru.EventCancelArgs
+			data, err := ioutil.ReadAll(req.Body)
+			c.Assert(err, check.IsNil)
+			err = json.Unmarshal(data, &eventCancel)
+			c.Assert(err, check.IsNil)
```

Comment:
@CarlosRoGuerra you could simply using json.NewDecoder, please read about this struct